game/core.py

```diff
@@ -37,6 +37,7 @@ def __init__(self, game_id):
         self.game_id = game_id
         self.players = []
         self.round_no = 1
+        self.num_AI = 0
```

Comment:
Why do you need this? You can always count by scrolling through all players and checking which of them are AI.

Reply:
Removed it.